docs/user_guides/single_table/predefined_constraints.rst

```diff
@@ -0,0 +1,410 @@
+**Constraints** that indicate **SDV** how to work with this data.
+
+UniqueCombinations Constraint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first constraint that we will explore is the ``UniqueCombinations``
+constraint.
+
+This Constraint class can handle the situation number 1 indicated above,
+in which the values of a set of columns can only be combined exactly as
+seen in the original data, and new combinations are not accepted. In
+order to use this constraint we will need to import it from the
+``sdv.constraints`` module and create an instance of it indicating:
+
+-  the names of the affected columns
+-  which strategy we want to use: ``transform`` or ``reject_sampling``
+
+.. ipython:: python
+    :okwarning:
+
+    from sdv.constraints import UniqueCombinations
+
+    unique_company_department_constraint = UniqueCombinations(
+        columns=['company', 'department'],
+        handling_strategy='transform'
+    )
+
+GreaterThan Constraint
+~~~~~~~~~~~~~~~~~~~~~~
+
+The second constraint that we need for our data is the ``GreaterThan``
+constraint. This constraint guarantees that one column is always greater
+than the other one. In order to use it, we need to create an instance
+passing:
+
+-  the name of the ``low`` column
+-  the name of the ``high`` column
+-  the handling strategy that we want to use
+
+.. ipython:: python
+    :okwarning:
+
+    from sdv.constraints import GreaterThan
+
+    age_gt_age_when_joined_constraint = GreaterThan(
+        low='age_when_joined',
+        high='age',
+        handling_strategy='reject_sampling'
+    )
+
+The ``GreaterThan`` constraint can also be used to guarantee a column is greater
+or lower than a scalar value or specific datetime value instead of another column. 
+To use this functionality, we can pass:
+
+-  the scalar value for ``low`` or the scalar value for ``high``
+-  a flag indicating whether ``low`` or ``high`` is a scalar
+
+.. ipython:: python
+    :okwarning:
+
+    salary_gt_30000_constraint = GreaterThan(
+        low=30000,
+        high='salary',
+        scalar='low',
+        handling_strategy='reject_sampling'
+    )
+
+.. note::
+    If you want to indicate that the column must be *lower than* a scalar value, 
+    all you need to do is invert the arguments, pass the scalar value as the ``high`` 
+    argument, the column name as the ``low`` argument, and set the `scalar` flag to ``"high"``.
+
+Optionally, when constructing ``GreaterThan`` constraint we can specify 
+more than a single column in either the ``high`` or ``low`` arguments. 
+For example, we can create a ``GreaterThan`` constraint that that ensures 
+that both the years in the company and prior years of experience is more 
+than one year.
+
+.. ipython:: python
+    :okwarning:
+
+    experience_years_gt_one_constraint = GreaterThan(
+        low=1,
+        high=['years_in_the_company', 'prior_years_experience'],
+        scalar='low',
+        handling_strategy='reject_sampling'
+    )
```

Comment:
I just realized that the training data contradicts this constraints because we have "years_in_the_company" equal to zero which mainly falls on my part. Maybe we can change this to `low=0`.

Reply:
Wouldn't you just use the `Positive` constraint then?

Reply:
Changed demo data to not generate 0s